recipes/libgpuarray/meta.yaml

```diff
@@ -0,0 +1,53 @@
+package:
+  name: libgpuarray
+  version: "0.6.0"
+
+source:
+  git_url: https://github.com/Theano/libgpuarray.git
+  git_tag: v0.6.0
```

Comment:
We prefer to use tarballs instead of cloning from VC. Is there a tarball we can use instead? We can always use the GitHub generated ones if nothing else.